### PR TITLE
Fix nbval testing failures due to new 0.10 release

### PR DESCRIPTION
**Pull request by pgbarletta**
From #248 and #249

- I filed an issue on nbval(https://github.com/computationalmodelling/nbval/issues/194) where I explain what changed.
For now, I removed all semicolons that were used to silence cell outputs and added the necessary additional sanitize statements on `tests/sanitize_output.cfg`
- Also unpinned nbval

One small drawback is that for now I can't silence this ugly plots:
ugly_plot

### CODE REVIEW

**Comment by pgbarletta** [comment]
Should I close this?

**Comment by IAlibay** [comment]
> Should I close this?

I have no context for this fix, is it still a necessary change in your opinion?

**Comment by pgbarletta** [comment]
Well, this was done to fix #248 and #249, and back then it worked (IDK why it's failing right now).
But then it seems that you pinned nbval on #250 and carried on.
I could rebase this branch, make everything pass again and unpin nbval if you want.

The nbval bug that caused this is still there by the way.

**Comment by IAlibay** [comment]
> Well, this was done to fix #248 and #249, and back then it worked (IDK why it's failing right now). But then it seems that you pinned nbval on #250 and carried on. I could rebase this branch, make everything pass again and unpin nbval if you want.
> 
> The nbval bug that caused this is still there by the way.

Ok, yeah it looks like this fix is still necessary. If you have the capacity to work out why things are failing now then that'd be great. Please go ahead and ping the whole coredevs group when you need a review (I don't spend a lot of time looking at this repo unfortunately).

(#250 was just a temporary fix so I could get a release out)

**Comment by pgbarletta** [comment]
@MDAnalysis/coredevs Done!

No big changes here. Just sanitizing notebooks* so nbval tests pass without the need of pinning it to an older version.

*: that is, adding regex expressions so outputs that necessarily vary from execution to execution (eg: data structures' memory locations) don't make nbval fail spuriously.

File: doc/source/examples/analysis/alignment_and_rms/aligning_trajectory_to_frame.ipynb
Comment (IAlibay) [inline doc/source/examples/analysis/alignment_and_rms/aligning_trajectory_to_frame.ipynb:1]: Why such a large delta here?

Comment (pgbarletta) [reply doc/source/examples/analysis/alignment_and_rms/aligning_trajectory_to_frame.ipynb:1]: again, accumulated widgets. That one was the heaviest.

**Review by lilyminium — CHANGES_REQUESTED** [review]
Thanks @pgbarletta for getting on this. Looks good, just a suggestion: `ax = ps.plot(...)` should still show the plot, I think, but with a much smaller output.
Screenshot 2023-06-10 at 3 41 35 pm

**Review by lilyminium — APPROVED** [review]
LGTM!

Sorry for all the conflicts...

**Comment by pgbarletta** [comment]
> LGTM!
> 
> Sorry for all the conflicts...

Yeah, do you know why they showed up all of a sudden?

**Comment by lilyminium** [comment]
I'm guessing because I merged #256 -- the widget states might be conflicting?

**Comment by pgbarletta** [comment]
ah, right, it's gotta be that.

**Comment by IAlibay** [comment]
Just saw the review request - apologies @pgbarletta I'm kinda checked out of userguide things for the time being. I'm leaving it up to @lilyminium to take this towards a merge.

**Comment by lilyminium** [comment]
@pgbarletta are you up for fixing the merge conflicts? Otherwise this PR is good to go :)

**Comment by pgbarletta** [comment]
Done! Removed all the widget attributes.

**Comment by lilyminium** [comment]
Thanks so much for this @pgbarletta!

**Pull request merged**